### PR TITLE
Adding totalSentiment()

### PR DESCRIPTION
**Pull request by mlkirby73**
Added the totalSentiment() method to Review.java. @MalaZerb @Parkertopak

### CODE REVIEW

File: Review.java [hunk 1]
```diff
@@ -165,4 +165,29 @@ public static String randomAdjective()
...
+     {
+      removePunctuation(word);
+      senValue += sentimentVal(word);
+      word = "";
+     }
+    }
+    return senValue;
+   }
+
 }
```
Comment (MalaZerb) [inline Review.java]: Everything looks great! Only recommendation is pseudo-code to briefly describe what is going on at each step, that's about it.

Comment (mlkirby73) [reply Review.java]: Thanks!

File: Review.java [hunk 2]
```diff
@@ -173,8 +173,10 @@ public static double totalSentiment(String fileName)
       String word = "";
       String reviewText = textToString(fileName);
       
+      // moves through the entire review
       for (int i = 0; i < reviewText.length(); i++)
       {
+         // checks if a word has been completed
```
Comment (andrewbonsted) [inline Review.java:179]: Great pseudocode, I can clearly see what is happening in this code!

**Comment by Parkertopak** [comment]
You did a great job writing the pseudocode and I can easily see what is going on in this code I would just add more comments so someone that doesnt know whats going on in the code can understand!

**Pull request merged**